### DISCUSSION TITLE
Unable to get 180° scene description

### DISCUSSION THREAD

**Discussion by Ragnarox97** [alicevision/Meshroom#1427] [Q&A] [answered]
Hi, I'm trying to scan a small object (an Iron Man action figure) by taking the photos using, as a background, a completely flattened black blanket and as clean as possible. I'm shooting in both JPG and RAW, using as shooting parameters:
ISO-100
f / 36
13 sec
200 mm

The camera is fixed and after each shot I rotate the model slightly. I am following this procedure https://peterfalkingham.com/2019/01/16/small-object-photogrammetry-how-to-take-photos/

The problem is that Meshroom cannot generate the cameras all around the mesh, but generates them by placing them in an arc. That is, it fails to understand that I am simulating a rotation around the object.

Modified Nodes
FEATURE EXTRACTION: akaze activated; describer density -> Ultra; describer quality -> Ultra
MESHING: min observation angle for SfM space estimation -> 1

![meshroom_screen](https://user-images.githubusercontent.com/74871201/119639725-2c7fd600-be18-11eb-8c28-e862e6064cab.JPG)

What could be the problem?

I upload the photos I used.
https://drive.google.com/file/d/1si-ZdIn-CvGL1-Ysq4EPaqHoHbV5RdSd/view?usp=sharing

This is just one example. I have also done others but I limit myself to sharing only this one. The others are all the same anyway.

**Comment by natowi** [comment, marked as answer]
Your camera probably still picks up some features from the background that looks plain black to you. Enable the FeatureViewer to see if that is the case (Similar to https://github.com/alicevision/meshroom/issues/1423)

**Reply by Ragnarox97** [reply]
Thank you for your answer. I checked the images with the "Display Feature" button and actually there are many features (orange dots) outside the main subject. In the post you linked to me you said you have to mask the subject. Is it an internal feature of Meshroom or do I just have to create a rough selection around the subject, in Photoshop, and fill it with black (or white)?

**Reply by natowi** [reply]
At the moment you need to manually cut out  the background  in Photoshop for example. This feature will be added in a future release.
